### ISSUE TITLE
Radio-Image not working

### ISSUE THREAD

**Issue by ryansmiller** [themeum/kirki#682]
I've been stuck trying to add images to the radio-image control. None of the src fields get any data when it's populated. Here's some example code:

```
Kirki::add_field( 'kirki_demo', array(
    'type'        => 'radio-image',
    'settings'    => 'radio_image_demo',
    'label'       => __( 'Radio-Image Control', 'kirki' ),
    'description' => __( 'This is a radio control that allows you to define an image for every option is the array of choices in the field. Useful if you want for example to select a layout. You can even use the images included in the Kirki plugin for that, in the /assets/images directory.', 'kirki-demo' ),
    'section'     => 'radio',
    'default'     => 'left',
    'priority'    => 10,
    'choices'     => array(
        'left'   => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
        'center' => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
        'right'  => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
    ),
) );
```

I've tried local files and images on other servers. Nothing seems to populate the data into the src field. Wondering if it's a problem with the array in the control file.

Btw, this is the HTML output I get:

```
<label>
                <input type="radio" value="left" name="_customize-radio-image-rock[radio_image_demo]" data-customize-setting-link="rock[radio_image_demo]" checked="checked">

                <span class="screen-reader-text"></span>

                <img src="" alt="">
            </label>
```

**Comment by aristath** [comment]
@ryansmiller I just copy-pasted your code in my test theme and it works fine...
Can you please post your config as well? (the `Kirki::add_config()` call you have)

**Comment by ryansmiller** [comment]
Hi @aristath yeah here's the code:

```
Kirki::add_config( 'rock', array(
    'capability'    => 'edit_theme_options',
    'option_type'   => 'option',
    'option_name'   => 'rock',
) );

Kirki::add_field( 'rock', array(
    'type'        => 'radio-image',
    'settings'    => 'radio_image_demo',
    'label'       => __( 'Radio-Image Control', 'rock' ),
    'description' => __( 'This is a radio control that allows you to define an image for every option is the array of choices in the field. Useful if you want for example to select a layout. You can even use the images included in the Kirki plugin for that, in the /assets/images directory.', 'rock' ),
    'section'     => 'header_layout',
    'default'     => 'left',
    'priority'    => 10,
    'choices'     => array(
        'left'   => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
        'center' => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
        'right'  => 'https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png',
    ),
) );
```

Here's a screenshot of what it looks like in the customizer.

![screen shot 2016-02-27 at 8 31 32 pm](https://cloud.githubusercontent.com/assets/11653521/13376705/1f9bf7ce-dd91-11e5-95ca-5a905bce0553.png)

**Comment by aristath** [comment]
ok, the issue only occurs when using options. It seems to be ok when using theme_mods which is what I'm always using so it never crossed my mind to check.

Marking this one as a bug and I'll take a closer look at it tomorrow :+1:

**Comment by ryansmiller** [comment]
Sweet! Thanks @aristath

**Comment by aristath** [comment]
@ryansmiller Can you please trying if this works using the [develop branch](https://github.com/aristath/kirki/tree/develop)?

I just tested it there and can't replicate the issue....
It still seems to be working fine! Perhaps this was an issue that has already been fixed there (after all, the develop branch is 320+ commits ahead of master)
The control is properly displayed, so I selected the 3rd image, savedd, and then I tested if they were properly saved by doing

``` php
var_dump( get_option( 'rock', array() );
```

The result was perfect:

```
array (size=1)
  'radio_image_demo' => string 'right' (length=5)
```

**Comment by aristath** [comment]
By the way, I set the develop branch to be the main branch in this repository.
Please try dowloading the new version and test if this has been resolved for you

**Comment by aristath** [comment]
This has been fixed. There was another issue with IE11 not working which has also been fixed on #674
